corehq/apps/app_manager/tests/test_dbaccessors.py

```diff
@@ -281,3 +282,9 @@ def test_get_apps_by_id(self):
     def test_get_latest_released_app_version(self):
         version = get_latest_released_app_version(self.domain, self.app_id)
         self.assertEqual(version, 2)
+
+    def test_get_latest_released_app_versions_by_app_id(self):
```

Comment:
to make this a real test there should be at least 2 released builds for the app so that you ensure you're getting the latest version

Reply:
a24aa485339